scripts/generate-javadoc.sh

```diff
@@ -14,7 +14,7 @@ mkdir_p $ARCHIVE_DIR
 function generate_javadoc_core() {
     declare release=$1
     # First we need to get the built javadocs. 
-    wget https://repo.jenkins-ci.org/releases/org/jenkins-ci/main/jenkins-core/${release}/jenkins-core-${release}-javadoc.jar
+    wget https://repo.jenkins-ci.org/public/org/jenkins-ci/main/jenkins-core/${release}/jenkins-core-${release}-javadoc.jar
```

Comment:
This one seems definitely wrong, as it's for core?

Reply:
It is not _necessary_ for purposes of this PR but I preferred to be consistent. Why do you say it is _wrong_?

Reply:
True, it's just unnecessary (and potentially more expensive on the Artifactory side) because not precisely looking at just our artifacts.